config/config.js

```diff
@@ -9,6 +9,7 @@ module.exports = {
   env: process.env.NODE_ENV || 'development',
   ci: process.env.CI,
   port,
+  contentStoreType: process.env.CONTENT_STORE_TYPE || 'file',
```

Comment:
Should this be part of the contentStore object in config? Or maybe we make that more specific to being an external API?